core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1037,18 +1037,10 @@ public MetricKey build() {
           .build();
 
   // Client metrics
-  public static final MetricKey CLIENT_BLOCK_READ_CHUNK =
-      new Builder("Client.BlockReadDataChunk")
+  public static final MetricKey CLIENT_BLOCK_READ_CHUNK_REMOTE =
+      new Builder("Client.BlockReadDataChunkRemote")
```

Comment:
`CLIENT_BLOCK_READ_CHUNK_REMOTE` is inconsistent with `Client.BlockReadDataChunkRemote`
I will leave this up to you which one to choose

Reply:
sorry will rebuilt from the master branch and redo the docGen